### ISSUE TITLE
Cannot hot reload updated atlas.

### ISSUE THREAD

**Issue by sicher** [defold/editor2-issues#1737]
STR:

1. Create atlas. Add image to sprite.
2. Run engine.
3. Change atlas and try hot reload it.

Actual result:

Does not reload (seems to try to reload atlas instead of texture and textureset):

```
ERROR:RESOURCE: /main/logo.atlasc could not be reloaded since it was never loaded before.
```

Expected result:

Works in editor 1:

```
INFO:RESOURCE: /main/logo.texturec was successfully reloaded.
INFO:RESOURCE: main/logo.texturesetc was successfully reloaded.
```

**Comment by AGulev** [comment]
When I try to reload atlas:

INFO:RESOURCE: /art/cards.texturesetc was successfully reloaded.
Assertion failed: (org_width > 0 && org_height > 0), function RenderBoxNodes, file ../src/gamesys/components/comp_gui.cpp, line 1067.
INFO:CRASH: Successfully wrote Crashdump to file: /Users/agulev/Library/Application Support/Defold/_crash

[_crash.zip](https://github.com/defold/editor2-issues/files/2730295/_crash.zip)


and 

INFO:RESOURCE: /art/cards.texturesetc was successfully reloaded.
ERROR:GRAPHICS: gl error 1280: invalid enumerant

Assertion failed: (0), function EnableTexture, file ../src/opengl/graphics_opengl.cpp, line 2178.
INFO:CRASH: Successfully wrote Crashdump to file: /Users/agulev/Library/Application Support/Defold/_crash

[_crash.zip](https://github.com/defold/editor2-issues/files/2730301/_crash.zip)

**Comment by AGulev** [comment]
Fixed in 1.2.147 
https://forum.defold.com/t/defold-1-2-147-has-been-released/43851
